extension/test/event-page-data-webtest.js

```diff
@@ -27,7 +27,8 @@ describe('EventPage', function() {
       ]);
       assert.isOk(result);
       assert.isOk(
-          result.hasOwnProperty('urla'),
+        // eslint-disable-next-line no-prototype-builtins
+        result.hasOwnProperty('urla'),
```

Comment:
indent appears wrong here and below.

Reply:
IMO, this is jumping about to prevent a problem that is vanishingly rare, and I would just skip it.
But if we ARE imposing this rule we should fix the code and not add exceptions. 
`result.hasOwnProperty(...)` => `Object.prototype.hasOwnProperty.call(result, ...)`